packages/kas/src/nodes.ts

```diff
@@ -3744,42 +3809,36 @@ var siPrefixes = {
 };
 
 // Use these two values to mark a unit as either SI-prefixable or not.
-var hasPrefixes = {};
-var hasntPrefixes = {};
```

Comment:
🤨 This is some big brain JS.

I think `Symbol` makes sense. Also wonder if a string literal would work?

```
const hasPrefixes = "HAS_PREFIXES";
const hasntPrefixes = "HASNT_PREFIXES";

type Prefixes = hasPrefixes | hasntPrefixes;
```

A possible nice side effect is that console logging would make it visible whether it's has/hasnt

Reply:
Strings would be easier.  I'm not sure why the original author tried to be fancy with `hasPRefixes = {}`.  I'll switch these to strings.